lms/djangoapps/grades/models.py

```diff
@@ -0,0 +1,42 @@
+from django.contrib.auth.models import User
+from django.db import models
+
+from xmodule_django.models import BlockTypeKeyField, CourseKeyField, UsageKeyField
+
+class Grade(models.Model):
+    """
+    A django model tracking persistent grades at the subsection level.
+    """
+
+    class Meta(object):
+        index_together = [
+            ['user_id', 'course_id', 'content_type', 'is_valid'],
+            ['course_id', 'updated'],
+            ['updated'],
+        ]
```

Comment:
Doesn't it make sense to add the additional indices only after we've determined how we'll need them?
Otherwise, they'll take up space and time.